src/Controller/ConnexionController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Form\LdapUNGType;
+use App\Kernel;
+use App\Service\JWTManagement;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use phpCAS;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Ldap\Ldap;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/v1/connexion", name="")
+ *
+ * Les routes présentes dans ce fichier doivent être sous la forme
+ * route : nomDuService
+ * name : nomDuService
+ *
+ * Class ConnexionController
+ */
+class ConnexionController extends AbstractController
+{
+    /**
+     * @Route("/cas", name="cas")
+     *
+     * @return RedirectResponse
+     */
+    public function cas(EntityManagerInterface $entityManager, JWTManagement $JWTManagement, Kernel $kernel)
+    {
+        phpCAS::client(
+            CAS_VERSION_2_0,
+            'cas.utt.fr',
+            443,
+            '/cas'
+        );
+        phpCAS::setCasServerCACert(
+            $kernel->getProjectDir().'/config/certificates/chain-cas-utt.pem',
+            true
+        );
+        phpCAS::forceAuthentication();
+        if (phpCAS::getUser()) {
+            if (!$entityManager->getRepository(User::class)->findOneBy(['casUid' => phpCAS::getUser()])) {
+                $user = new User();
+                $user->setCasUid(phpCAS::getUser());
+                $entityManager->persist($user);
+                $entityManager->flush();
+            } else {
+                $user = $entityManager->getRepository(User::class)->findOneBy(['casUid' => phpCAS::getUser()]);
+            }
+
+            return $this->redirect($JWTManagement->getFrontURLFromUser($user));
+        }
+
+        throw $this->createAccessDeniedException('Auth cas failed !');
+    }
+
+    /**
+     * @Route("/ldapUNG", name="ldapUNG")
+     */
+    public function ldapUNG(Request $request, EntityManagerInterface $entityManager, JWTManagement $JWTManagement)
+    {
+        $form = $this->createForm(LdapUNGType::class);
+        //$form->handleRequest($request);
+        $form->submit($request->request->all());
```

Comment:
why not use the handleRequest method commented above?
```suggestion
        $form->handleRequest($request);
```

Reply:
it doesn't work, still trying to find out why...

Reply:
Does your code work though? If so, I think it can be left as-is, and I’ll try to open a PR for that as soon as I can

Reply:
yes it works, and I found a real solution !

Symfony considers a form as submitted when the fields are like this : form_name[field], when I was using field directly.

Reply:
Oh great! 🎉